esp-hal/src/gpio/mod.rs

```diff
@@ -2513,6 +2552,7 @@ mod embedded_hal_impls {
         }
     }
 
+    #[instability::unstable]
```

Comment:
oh - totally missed we can now use the macro on impl blocks 👍

Reply:
yes but some care must be taken because the whole impl block is deleted without the unstable feature